tests/test_rocket.py

```diff
@@ -91,8 +91,6 @@ def mainTrigger(p, h, y):
     assert FinSet.allInfo() == None
     # Check if Tail allInfo() is working properly
     assert Tail.allInfo() == None
-    # Check if draw method is working properly
-    assert FinSet.draw() == None
```

Comment:
Why remove this?

Reply:
The Fin.draw() method doesn't exist anymore. Only the Fin.plots.draw()

Reply:
If you really want to, we can create a draw() method inside each of the Fins classes, which will return self.plots.draw(), but I honestly don't see how that could be helpful.